app/services/partner_invite_service.rb

```diff
@@ -9,7 +9,10 @@ def call
     return self unless valid?
 
     partner.update!(status: 'invited')
-    User.invite!(email: partner.email, partner: partner.profile)
+    user = User.invite!(email: partner.email, partner: partner.profile, skip_invitation: true)
+
+    user.reload
+    user.deliver_invitation
```

Comment:
One minor suggestion, can you add a comment to explain why this was added? I sense it will be unclear to future maintainers about why this was added.

Reply:
Hi @edwinthinks,  I  agree. Is it ok? Thanks